### PR TITLE
Issue/119: Add docstrings

### PR DESCRIPTION
**Pull request by m3nu**

### CODE REVIEW

File: invoice2data/extract/invoice_template.py
Comment (m3nu) [inline invoice2data/extract/invoice_template.py]: The `self` argument is passed automatically and shouldn't be included in the docstring.

Comment (duskybomb) [reply invoice2data/extract/invoice_template.py]: oh, thanks

Comment (duskybomb) [reply invoice2data/extract/invoice_template.py]: Should I remove it or you are?

File: invoice2data/extract/loader.py
Comment (m3nu) [inline invoice2data/extract/loader.py]: This won't work because Python doesn't expand the `~` char. It also doesn't make sense because you already included your home folder.

Comment (m3nu) [reply invoice2data/extract/loader.py]: Since just reading the some templates isn't useful, it would be good to show what to do with them later.

File: invoice2data/extract/invoice_template.py
Comment (m3nu) [reply invoice2data/extract/invoice_template.py]: Feel free to update it yourself. I'm working on Factur-X today.

**Comment by m3nu** [comment]
Thanks for the update. Now we wait for those test to pass and then merge it.

**Comment by duskybomb** [comment]
I guess, I will have to review it before merging because you made the PR

**Review by duskybomb — APPROVED** [review]

**Pull request merged**
